superset/translations/sl/LC_MESSAGES/messages.json

```diff
@@ -81,75 +111,114 @@
+      "1 calendar day frequency": ["frekvenca: 1 koledarski dan"],
+      "1 day": ["1 dan"],
+      "1 day ago": ["1 day ago"],
       "1 hour": ["1 ura"],
+      "1 hourly frequency": ["frekvenca: 1 ura"],
       "1 minute": ["1 minuta"],
-      "1 minutely frequency": [""],
-      "1 month end frequency": [""],
-      "1 month start frequency": [""],
-      "1 week ago": [""],
-      "1 year ago": [""],
-      "10 minute": ["10 minut"],
-      "104 weeks ago": [""],
-      "15 minute": ["15 minut"],
-      "156 weeks ago": [""],
-      "1AS": [""],
-      "1D": [""],
-      "1H": [""],
-      "1M": [""],
-      "1T": [""],
-      "2 years ago": [""],
-      "2/98 percentiles": [""],
-      "28 days ago": [""],
+      "1 minutely frequency": ["frekvenca: 1 minuta"],
+      "1 month end frequency": ["frekvenca: 1 mesec - konec"],
+      "1 month start frequency": ["frekvenca: 1 mesec - začetek"],
+      "1 week": ["1 week"],
+      "1 week ago": ["1 week ago"],
+      "1 week starting Monday (freq=W-MON)": [
+        "1 teden z začetkom v ponedeljek (freq=W-MON)"
+      ],
+      "1 week starting Sunday (freq=W-SUN)": [
+        "1 teden z začetkom v nedeljo (freq=W-SUN)"
+      ],
+      "1 year": ["1 year"],
+      "1 year ago": ["1 year ago"],
+      "1 year end frequency": ["frekvenca: 1 leto - konec"],
+      "1 year start frequency": ["frekvenca: 1 leto - začetek"],
+      "10 minute": ["10 minute"],
+      "104 weeks": ["104 weeks"],
+      "104 weeks ago": ["104 weeks ago"],
+      "15 minute": ["15 minute"],
+      "156 weeks": ["156 weeks"],
+      "156 weeks ago": ["156 weeks ago"],
+      "1AS": ["1AS"],
+      "1D": ["1D"],
+      "1H": ["1H"],
+      "1M": ["1M"],
+      "1T": ["1T"],
+      "2 years": ["2 years"],
+      "2 years ago": ["2 years ago"],
+      "2/98 percentiles": ["2/98 percentil"],
+      "28 days": ["28 days"],
+      "28 days ago": ["28 days ago"],
       "2D": ["2D"],
       "3 letter code of the country": ["Tričrkovna oznaka države"],
-      "3 years ago": [""],
+      "3 years": ["3 years"],
+      "3 years ago": ["3 years ago"],
       "30 days": ["30 dni"],
+      "30 days ago": ["30 days ago"],
       "30 minute": ["30 minut"],
-      "30 minutes": ["30 minut"],
-      "30 second": ["30 sekund"],
-      "30 seconds": ["30 sekund"],
+      "30 minutes": ["30 minutes"],
+      "30 second": ["30 second"],
+      "30 seconds": ["30 seconds"],
       "3D": ["3D"],
-      "4 weeks (freq=4W-MON)": [""],
-      "5 minute": ["5 minut"],
-      "5 minutes": ["5 minut"],
-      "5 second": ["5 sekund"],
-      "52 weeks ago": [""],
-      "6 hour": ["6 ur"],
-      "60 days": ["60 dni"],
-      "7 calendar day frequency": [""],
-      "7D": [""],
-      "9/91 percentiles": [""],
-      "90 days": ["90 dni"],
+      "4 weeks (freq=4W-MON)": ["4 tedni (freq=4W-MON)"],
+      "5 minute": ["5 minute"],
+      "5 minutes": ["5 minutes"],
+      "5 second": ["5 second"],
+      "5 seconds": ["5 seconds"],
```

Comment:
Why do so many strings in this PR seem like they're being translated back into English?

Reply:
Correct me if I'm wrong, but these are mostly time deltas for chart Advanced analytics which supports natural language (ISO 8601 durations) and time filter granularities which support natural language also.

I thought it would be confusing for users if these are translated, because natural language input has to be in English and these strings also serve as input examples. That's why I left those in English.

(Maybe I made some mistakes also. Sometimes it is hard to find translation string context, there are almost 4 thousand strings)